acct-user/sndiod/sndiod-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="daemon user for sndio"
+ACCT_USER_ID="-1"
+ACCT_USER_GROUPS=( audio )
+ACCT_USER_HOME=/var/lib/sndio
```

Comment:
See [this](http://michael.orlitzky.com/articles/gentoo_glep81_user_package_guidelines.xhtml) just to check you need USER_HOME.